tests/action/test_ammo_use.py

```diff
@@ -85,9 +90,11 @@ def test_ammo_fire_all(self):
         for ent_id in self.ammo })
 
     # check if the ammos were consumed
+    ammo_ids = []
     for ent_id in self.ammo:
       self.assertEqual(1, # True
         ItemState.parse_array(env.obs[ent_id].inventory.values[0]).quantity)
```

Comment:
can we just item = parse_array..... so that we only do this once? NBD

Reply:
Right, we only need to parse_array once to get both quantity and ids. Getting to know better about the datastore interface ha